profiler/top_profiler.py

```diff
@@ -0,0 +1,25 @@
+import data
+
+# Processes the set of previously written logs
+
+threshold = 1.10  # Percent difference
+
+db = data.FileBasedProfileDB()
+actual_max_ts = 0
+dt = db.find("python ann_data.py")
+last_two = dt[-2:]
+c = 0
+
+for s in dt:
```

Comment:
```suggestion
for s in last_two:
```
I _think_ this is the intention...